backend/src/entities/contraceptive.entity.ts

```diff
@@ -0,0 +1,85 @@
+import { Entity, Column, PrimaryGeneratedColumn, OneToMany } from 'typeorm';
+import { Benefit } from './benefits.entity';
+import { SideEffect } from './side-effects.entity';
+import { Tag } from './tags.entity';
+import { ThingToKnow } from './things-to-know.entity';
+
+type TimeUnits = 'years' | 'months' | 'days' | 'weeks';
+
+@Entity()
+export class Contraceptive {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column()
+  usePatternLowBound: number;
+
+  @Column()
+  usePatternHighBound: number;
+
+  @Column()
+  usePatternUnits: TimeUnits;
+
+  @Column()
+  effectiveRate: number;
+
+  @Column()
+  costMin: number;
+
+  @Column()
+  costMax: number;
+
+  @Column()
+  accessibility: string;
+
+  @OneToMany((type) => Tag, (tag) => tag.id)
+  tags: Tag[];
+
+  @Column()
+  description: string;
+
+  @Column()
+  use: string;
+
+  @Column()
+  inCaseOfProblem: string;
+
+  @Column({ type: 'text', array: true, default: [] })
+  whenItStartsToWork: string[];
```

Comment:
Can we make this a string[][]? Same styling requirement as healthRisks

Reply:
The problem we're trying to solve is determining how to represent the text, such that we can capture the information of the "bolded" text

Any suggestions? string[][] would work, its a little hacky LOL but it works

Reply:
Kevin ended up abstracting the highlighting I used for the health risks section so that it works with the bolded text in this section as well, and I think to accommodate that he set it up so that each bullet is a string[] and then the whole list is a string[][].

Reply:
I think we can add the `<b>` tag in the text in our database instead of using this weird string[][] format. That way we could use a `<pre>` tag (or something memory is not 100%) but I am completely opposed to the string[][] hack. 

Additionally, I don't think I _can_ have a string[][] in the db